README.md

```diff
@@ -77,6 +79,56 @@ release of Alda.
 
 [alda-install]: https://alda.io/install
 
+## Building from Source
+
+If pre-built binaries are not available for your system (e.g., Raspberry Pi,
+BSD), you can build Alda from source. Alda consists of two components: the
+**client** (written in Go) and the **player** (written in Kotlin/Java).
```

Comment:
One correction: builds actually **are** available now for Raspberry Pi (Linux ARM) on the install page, which is a recent addition. So we can leave out the mention of Raspberry Pi here.

Maybe we can simplify it like this:

```suggestion
Another option is to build Alda from source. Alda consists of two components:
the **client** (written in Go) and the **player** (written in Kotlin/Java).
```